build/azDevOps/azure/agent-config-vars.yml

```diff
@@ -1,10 +1,4 @@
 variables:
-
   # Agent configuration
   - name: pool_vm_image
     value: ubuntu-22.04
-  - name: TaskctlVersion
-    value: 1.4.2
-  - name: StacksEnvfileVersion
-    value: 0.0.16
```

Comment:
Why are these being removed?

Reply:
I believe it was because taskCtlVersion was already being defined on setup.yml

Reply:
@zecarrera I've pushed a change to standardise the passing of the version and put it back into the `agent-config-vars` file